client/src/app/learn-page/learn-page.component.ts

```diff
@@ -1,7 +1,7 @@
 import { Component, OnDestroy, OnInit } from '@angular/core';
 import { RecommenderService } from '../recommender.service';
 import { ActivatedRoute } from '@angular/router';
-import { Subscription } from 'rxjs';
+import {combineLatest, forkJoin, Subscription} from 'rxjs';
```

Comment:
did you install the precommit hooks? looks wrongly formatted.

Reply:
Of course I did, not. :) Will format